string_lab.js

```diff
@@ -0,0 +1,197 @@
+// # String Exercises
```

Comment:
Please revisit this assignment and correct comments provided below. Next time, please read each problem more carefully, draw mental map of how to solve the problem, and think about which string methods/properties would be appropriate to the problem. Keep practicing and asking for help.